### PR TITLE
added the details of the Icp projects

### PR DESCRIPTION
**Pull request by harshitchugh001**

### CODE REVIEW

File: data/ecosystems/i/internet-computer.toml [hunk 1]
```diff
@@ -21728,6 +21728,279 @@ url = "https://github.com/NnsDao/nnsdao-fe"
...
+url = "https://github.com/Chiksinafees/DKeeper-web3_project"
+
+[[repo]]
+url = "https://github.com/ManNjoro/icpcalculator"
+
+[[repo]]
+url = "https://github.com/Shubhamsg7/ar-icp-app"
+
+[[repo]]
+url = "https://github.com/Shubhamsg7/crypto-app"
```
Comment (eherrerosj) [inline data/ecosystems/i/internet-computer.toml:21828]: how's this using ICP?

**Review by eherrerosj — CHANGES_REQUESTED** [review]
left a couple comments

File: data/ecosystems/i/internet-computer.toml [hunk 2]
```diff
@@ -21728,6 +21728,279 @@ url = "https://github.com/NnsDao/nnsdao-fe"
...
+url = "https://github.com/Okonu/motoko"
+
+[[repo]]
+url = "https://github.com/abhishekpandit5599/react-native-research-fix-package"
+
+[[repo]]
+url = "https://github.com/abhishekpandit5599/motoko-db-research"
+
+[[repo]]
+url = "https://github.com/sidhant-shr/Xplore"
```
[the remark this answers is not in the document]
Comment (harshitchugh001) [reply data/ecosystems/i/internet-computer.toml]: Thank you for pointing out these repos , i have removed the requested repos and checked again for more such repos.
Hope this helps .

**Review by eherrerosj — APPROVED** [review]
thanks!

**Pull request merged**